docs/HANDLED_RULES.md

```diff
@@ -343,6 +344,21 @@ class SelfAssignement {
 
 -----
 
+#### "Thread.run" should not be called directly ([Sonar Rule 1217](https://rules.sonarsource.com/java/type/Bug/RSPEC-1217))
+
+The purpose of the Thread.run() method is to execute code in a separate, dedicated thread. Calling this method directly doesn't make sense because it causes its code to be executed in the current thread.
+
+To get the expected behavior, call the Thread.start() method instead.
```

Comment:
This is copied verbatim from https://rules.sonarsource.com/java/type/Bug/RSPEC-1217, which violates SonarSource's copyright of said material. Write what the processor does instead.